utils/report.c

```diff
@@ -82,6 +84,96 @@ static struct uftrace_report_node *find_or_create_node(struct rb_root *root, con
 	return node;
 }
 
+static struct uftrace_report_call *find_or_create_report_call(struct uftrace_fstack *fstack)
+{
+	struct uftrace_report_call *report_call;
+
+	list_for_each_entry(report_call, &report_calls, list) {
+		if (report_call->addr == fstack->addr)
+			return report_call;
+	}
+
+	report_call = malloc(sizeof(*report_call));
+	report_call->addr = fstack->addr;
+	report_call->callers_size = 0;
+	report_call->callees_size = 0;
+	INIT_LIST_HEAD(&report_call->callers);
+	INIT_LIST_HEAD(&report_call->callees);
```

Comment:
Maybe we can use arrays rather than list.

Reply:
@namhyung Then is it okay to set caller and callee array size to 1024?

Reply:
I set caller and callee array size to 1024.